pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -93,11 +93,19 @@ var (
 			Buckets: []float64{60, 3600, 7200},
 		},
 	)
+	provisionJobsDurationHistogram = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "provision_jobs_duration_seconds",
+			Help:    "Provision Jobs duration distribution",
+			Buckets: []float64{60, 3600, 7200},
```

Comment:
For install buckets could be a little diff, could you adjust these for 1m, 10m, 30m, 60m